pkgs/by-name/pk/pkl/package.nix

```diff
@@ -0,0 +1,77 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  gradle,
+  temurin-bin-21,
+  kotlin,
+  nix-update-script,
+}:
+let
+  jdk = temurin-bin-21;
+  gradleOverlay = gradle.override { java = jdk; };
+  kotlinOverlay = kotlin.override { jre = jdk; };
+  commitHash = "8bd738e";
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "pkl";
+  version = "0.28.2";
+
+  src = fetchFromGitHub {
+    owner = "apple";
+    repo = "pkl";
+    tag = finalAttrs.version;
+    hash = "sha256-+ppGy3MSqxtecMXVhF13OY5VZ3F2C38AhBgUufiykOU=";
+  };
+
+  nativeBuildInputs = [
+    gradleOverlay
+    jdk
+    kotlinOverlay
+  ];
+
+  mitmCache = gradle.fetchDeps {
+    inherit (finalAttrs) pname;
+    data = ./deps.json;
+  };
+
+  __darwinAllowLocalNetworking = true;
+
+  gradleFlags = [
+    "-x"
+    "spotlessCheck"
+    "-DreleaseBuild=true"
+    "-Dorg.gradle.java.home=${jdk}"
+    "-DcommitId=${commitHash}"
+    "-Porg.gradle.java.installations.auto-download=false"
+    "-Porg.gradle.java.installations.auto-detect=false"
+  ];
+
+  JAVA_TOOL_OPTIONS = "-Dfile.encoding=utf-8";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p "$out/bin"
+    head -n2 ./pkl-cli/build/executable/jpkl | sed 's%java%${jdk}/bin/java%' > "$out/bin/pkl"
+    tail -n+3 ./pkl-cli/build/executable/jpkl >> "$out/bin/pkl"
+    chmod 755 "$out/bin/pkl"
+
+    runHook postInstall
+  '';
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "A configuration as code language with rich validation and tooling.";
```

Comment:
This one slipped through: I think @emaryn wanted you to change this description.

Reply:
Yeah but it doesn't make sense to remove "a".